microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import Path
+
+from lisa import Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.features import Gpu, SerialConsole
+from lisa.features.gpu import ComputeSDK
+from lisa.operating_system import Redhat
+from lisa.sut_orchestrator.azure.common import AZURE, AzureNodeSchema
+from lisa.sut_orchestrator.azure.tools import LisDriver
+from lisa.testsuite import simple_requirement
+from lisa.tools import Lsmod
+from lisa.util import LisaException, SkippedException
+from lisa.util.perf_timer import create_timer
+
+
+@TestSuiteMetadata(
+    area="gpu",
+    category="functional",
+    description="""
+    This test suite runs the gpu test cases.
+    """,
+    tags=[],
+)
+class gpu(TestSuite):
+    grid_skus = ["Standard_NV"]
```

Comment:
CUDA and GRID are compute packages, they are not must have on a GPU vm size. Call them like NVIDIA_SKUs and AMD_SKUs.

Reply:
Grid and CUDA are both NVIDIA. I can rename it grid_supported_skus and cuda_supported_skus.